Ago-Dic-2020/rodriguez-martinez-diego-jose/Practica 1/matrix.py

```diff
@@ -0,0 +1,27 @@
+class Matrix:
+    matrix_List = []
+
+    def __init__(self, matrix_string):
```

Comment:
Puedes utilizar argumentos default 😉 -> https://www.geeksforgeeks.org/default-arguments-in-python/

Reply:
Para que no truene en caso de no pasarle un matrix_string? o de que serviria en este problema?

Reply:
Si, para que no haya conflicto en caso de que no le pases algún parámetro a la función y sea siempre de tipo string. También puedes utilizar [type-hinting](https://docs.python.org/3/library/typing.html), introducido al lenguaje en la version `3.5`, pero no es tan necesario con la metodología [duck-typing](https://realpython.com/lessons/duck-typing/) de python :wink: